ptsprojects/mynewt/gap_wid.py

```diff
@@ -37,6 +37,26 @@ def gap_wid_hdl(wid, description, test_case_name):
         return gen_wid_hdl(wid, description, test_case_name, False)
 
 
+def gap_wid_hdl_failed_enc(wid, description, test_case_name):
```

Comment:
I'm not sure but maybe this also should be moved to wid/gap? or is it platform specific?